### ISSUE TITLE
Improve aws-cognito performance

### ISSUE THREAD

**Issue by engharb** [pmill/aws-cognito#5]
Hello @pmill ;

I want to suggest the following improvements :

1. improve verifyAccessToken(): to return the payload completely not only the payload[username], this will help in getting more details about the logged in user like email address ...

2. improve downloadJwtWebKeys(): to return jwtWebKeys. that helps in storing the keys once locally or in our web server and re-use it (instead of caching). And new setter/getter functions for jwtWebKeys may be required for that.

Regards.

**Comment by pmill** [comment]
Hello,

For #1 the token doesn't actually include user details, only the username, the standard jwt claims, and a few others (event_id, token_use, client_id).

**Comment by pmill** [comment]
Hello, your second issue should be resolved now with the v0.2.0 release.

**Comment by engharb** [comment]
Hello,
regarding 

1. improve verifyAccessToken(): to return the payload completely not only the payload[username], this will help in getting more details about the logged in user like email address ...

As I told you it is necessary to return complete payload (where it contains some details like email of logged in user), I used another library from Jose for that reason. So if you are able to fix this I will never need for library of Jose and convert completely to use your API.

I need the email to register my users details in my backend server. I checked that $jwtPayload by dumping it and i used to see user's details i.e. email-address.

regards,
